src/routes/safe/components/Settings/ThresholdSettings/ChangeThreshold/index.tsx

```diff
@@ -88,11 +59,10 @@ export const ChangeThresholdModal = ({
 
   const handleThreshold = ({ target }) => {
     const value = parseInt(target.value)
-    setDisabledSubmitForm(value === editedThreshold || value === threshold)
     setEditedThreshold(value)
   }
 
-  const handleSubmit = ({ txParameters }) => {
+  const handleSubmit = (txParameters) => {
```

Comment:
Why change it from an object? Does it not make sense to keep it with keys so it is more readable?

Reply:
To follow the `onSubmit` type from `<TxParamsState />`
https://github.com/gnosis/safe-react/blob/cf92e1f86bc9783c26827789f54d59c643268fdb/src/routes/safe/components/Transactions/helpers/TxParamsState/index.tsx#L25